### DISCUSSION TITLE
Rename Deployment Environments on GitHub

### DISCUSSION THREAD

**Discussion by userari** [vercel/community#4636] [Help]
### Summary

Hi everyone,

I am wondering if there is a way to rename the Deployment Environments generated on GitHub when we deploy a Next.js app on Vercel ?
Such as " Production - project-doc " & " Production - project-web"

Because I am using a Turborepo, and when my projects are deployed , I have 2 productions deployments with 2 different names, but I want to rename them if possible.

Thanks in advanced ! 

### Steps to Reproduce

Deploy a Next.js App on Vercel.

**Comment by kcirtaptrick** [comment]
We have a monorepo that includes our backend with an existing naming convention. We have a backend engineer who doesn't like Vercel and will take every opportunity to criticize, it would be nice to have this feature to not give him another reason to.
